pl_bolts/datasets/sr_celeba_dataset.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+from pl_bolts.datasets.sr_dataset_mixin import SRDatasetMixin
+from pl_bolts.utils import _PIL_AVAILABLE, _TORCHVISION_AVAILABLE
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+if _PIL_AVAILABLE:
+    import PIL
```

Comment:
Not a strong opinion, but I prefer importing only the item needed in this module...
```suggestion
    from PIL import Image
```